src/components/Pill/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const getPills = (statuses) => {
+  const pills = [];
+  statuses.forEach((status) => {
+    switch (status) {
+      case 'unpublished':
+        pills.push({
+          text: 'Incomplete',
+          className: 'badge badge-danger',
+        });
+        break;
+      case 'reviewed':
+        pills.push({
+          text: 'Scheduled',
+          className: 'badge badge-primary',
+        });
+        break;
+      case 'review_by_legal':
+      case 'review_by_internal':
+        pills.push({
+          text: 'In review',
+          className: 'badge badge-warning',
+        });
+        pills.push({
+          text: <i className="fa fa-lock" />,
+        });
```

Comment:
Since lock state is global, maybe this is better done in the message at the top of the page that we plan to add?

Doesn't hurt to have it here too. We can always adjust later.

Reply:
I agree with that and will leave a comment in the ticket that handles the page level status messages.

I think it's nice to still leave in here so you know which course run(s) are causing you to be in review.